Terminal.Gui/View/ViewDrawing.cs

```diff
@@ -503,5 +503,25 @@ public virtual void OnDrawContentComplete (Rect contentArea)
 			DrawContentComplete?.Invoke (this, new DrawEventArgs (contentArea));
 		}
 
+		/// <summary>
+		/// Draw a frame based on the passed bounds to the screen relative.
+		/// </summary>
+		/// <param name="bounds">The bounds view relative.</param>
```

Comment:
I think this param should be named `rect` and the docs should read "The view relative location and size of the frame."